packages/react-core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -22,7 +22,7 @@ export interface DatePickerProps
    * menuAppendTo={() => document.body};
    * menuAppendTo={document.getElementById('target')}
    */
-  appendTo?: HTMLElement | ((ref?: HTMLElement) => HTMLElement) | 'parent';
+  appendTo?: HTMLElement | ((ref?: HTMLElement) => HTMLElement) | 'inline';
```

Comment:
need to change the jsdoc default as well

Reply:
Good catch!